packages/libs/deer/LICENSE-MIT

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2022 HASH
+Copyright (c) 2022 Bilal Mahmoud
```

Comment:
@nonparibus: Is the license ok that way? I was unsure how to do the copyright in the `LICENSE-MIT` and `LICENSE-APACHE` files. I would love your input.

Reply:
Direct-committed our MIT and Apache files (adding `.md` extensions in line with the repo style, as well). Happy for this to be dual-licensed under both sets of terms (our first time doing so!)